.github/workflows/tests-01-main.yml

```
@@ -51,7 +51,16 @@ jobs:
                 echo "Failed to detect ${APP} as installed" >&2
                 exit 1
               else
-                ./deb-get purge "${APP}"
+                PUBLISHED_VER=$(./deb-get show "${APP}" | grep "Published:" | grep -oE '[^[:space:]]+$' || :)
+                INSTALLED_VER=$(./deb-get show "${APP}" | grep "Installed:" | grep -oE '[^[:space:]]+$' || :)
```

This might be preferable - just a thought..
```suggestion
                INSTALLED_VER=$(dpkg-query -Wf '${Version}' "${APP}" || :)
```
Marginally more efficient and direct than running `./deb-get` again just to get it to do the above, wrap it in some text and then parse it out again.